tcms/testcases/static/testcases/js/search.js

```diff
@@ -118,6 +118,37 @@ $(document).ready(function() {
 
     hookIntoPagination('#resultsTable', table);
 
+    const selectAllButton = $("#check-all")
+
+    selectAllButton.click(function () {
+        const rowCheckboxInputButton = $("input:checkbox[name='row-check']")
+        if (selectAllButton.prop("checked")) {
+            rowCheckboxInputButton.prop("checked", true);
+            table.rows().select()
+        } else {
+            rowCheckboxInputButton.prop("checked", false);
+            table.rows().deselect()
```

Comment:
Ideally this will be 
```
value = selectAllButton.prop("checked");
rowCheckboxInputButton.prop("checked", value);
table.rows().XXXXX(value) // <- toggles between selected state
```

But if there's no easy way of changing state for all rows then don't bother with it.

Reply:
I think `value ? table.rows().select() : table.rows().deselect()` works